File: backend/FunnyMovies/FunnyMovies/Models/Vote.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace FunnyMovies.Models
{
	public class Vote
	{
		public long UserId { get; set; }
		public long MovieId { get; set; }
		public bool IsUpVote { get; set; }
	}
}

File: frontend/src/models/Vote.ts
import { observable, action, toJS, computed } from "mobx";
export class Vote{
    @observable userId: number;
    @observable movieId: number;
    @observable isUpVote: boolean|null;
    constructor(data?: any) {
        this.userId = -1;
        this.movieId=-1;
        this.isUpVote = null;
    }
}